terasoluna-gfw-parent/pom.xml

```diff
@@ -571,10 +571,10 @@
     <tiles.version>3.0.8</tiles.version>
     <!-- == Bouncy Castle == -->
     <bouncycastle.version>1.64</bouncycastle.version>
-    <!-- == Other JavaEE/SE APIs == -->
-    <javax-el.version>3.0.1-b06</javax-el.version>
-    <javax-inject.version>1</javax-inject.version>
-    <javax-jsp.version>2.3.3</javax-jsp.version>
+    <!-- == Other JakartaEE/SE APIs == -->
```

Comment:
- [ ] I don't know Jakarta SE by all means. Does it mean Java SE?

Reply:
Since Jakarta SE does not exist, this fix was incorrect. Changed the section name of the library group to something appropriate.
https://github.com/terasolunaorg/terasoluna-gfw/pull/1039/commits/e1126a3b05c96c8b339e150772c4066c26e9acc3